### ISSUE TITLE
Reset focus state after page navigation

### ISSUE THREAD

**Issue by schalkneethling** [mdn/mdn-fiori#86]
One of the side effects of using client-side navigation is that the header does not change between page loads(as we are just swapping in new content). This means that if your focus was on the "References & Guides" link in the header, it will still be there once the new page content loads.

For people of assistive technologies, this is not an ideal state. They now need to tab in reverse in an attempt to get back to the Skip to... links that are located earlier in the DOM. A solution to this might be to simply move focus to an item before the first skip to link, or perhaps move focus back to the window(window.focus) or remove focus from the current activeElement.

The reason we cannot simply move focus to the first skip to link is that it will then be visible to all users, irrespective of their use case.

Will need to test the various options and use the one that works most predictably between browsers and platforms.

## Bugzilla

https://bugzilla.mozilla.org/show_bug.cgi?id=1563471
